src/lib.rs

```diff
@@ -635,6 +635,21 @@ impl FromStr for LevelFilter {
     }
 }
 
+impl From<LevelFilter> for Option<Level> {
+    fn from(level: LevelFilter) -> Option<Level> {
+        level.to_level()
+    }
+}
+
+impl From<Option<Level>> for LevelFilter {
+    fn from(level: Option<Level>) -> LevelFilter {
+        match level {
+            Some(level) => level.to_level_filter(),
+            None => LevelFilter::Off,
```

Comment:
I'm sure this is always going to be obvious.

Reply:
Did you leave out "not" or are you disagreeing with the comment above here and saying that this is actually fine and a reasonable choice to convert `Option<Level>::None` to `LevelFilter::Off`?